pkg/tsdb/cloudwatch/utils_test.go

```diff
@@ -190,6 +191,15 @@ func newTestConfig() *setting.Cfg {
 	return &setting.Cfg{AWSAllowedAuthProviders: []string{"default"}, AWSAssumeRoleEnabled: true, AWSListMetricsPageLimit: 1000}
 }
 
+type mockSessionCache struct {
```

Comment:
We already have a `fakeSessionCache` in this file. Can you use it instead?

Reply:
Same as above. I'm using some assertions from the testify/mock library on both the session cache and the metrics client in order to try to prove that queries are grouped as regions.